aten/src/ATen/ParallelThreadPoolNative.cpp

```diff
@@ -67,10 +68,19 @@ int get_num_interop_threads() {
 }
 
 void launch(std::function<void()> func) {
+  auto fn = std::bind([](
```

Comment:
I'm surprised about the use of `std::bind` here. Normally a lambda is enough. What's going on here?

Reply:
move capture is not supported till C++14